CONTRIBUTION.md

```diff
@@ -1,14 +1,51 @@
-# Contribution:
+# Contribution Guidelines
 
-## Running the project:
+Hello :wave: and welcome to **דאטאבוס**, a project of [hasadna](https://open-bus-map-search.hasadna.org.il)!
 
-- fork the repo
-- clone the repo
-- `yarn install`
-- `yarn start`
```

Comment:
don't we want that?

Reply:
We have fork and clone instructions at following section